airbyte-cdk/python/unit_tests/sources/file_based/scenarios/csv_scenarios.py

```diff
@@ -34,6 +35,158 @@
         }
     )
     .set_file_type("csv")
+    .set_expected_spec(
+        {
+            "documentationUrl": "https://docs.airbyte.com/integrations/sources/in_memory_files",
+            "connectionSpecification": {
+                "title": "InMemorySpec",
+                "description": "Used during spec; allows the developer to configure the cloud provider specific options\nthat are needed "
+                               "when users configure a file-based source.",
+                "type": "object",
+                "properties": {
+                    "streams": {
+                        "title": "The list of streams to sync",
+                        "description": "Streams defines the behavior for grouping files together that will be synced to the downstream "
+                                       "destination. Each stream has it own independent configuration to handle which files to sync, "
+                                       "how files should be parsed, and the validation of records against the schema.",
+                        "order": 10,
+                        "type": "array",
+                        "items": {
+                            "title": "FileBasedStreamConfig",
+                            "type": "object",
+                            "properties": {
+                                "name": {
+                                    "title": "Name",
+                                    "type": "string"
+                                },
+                                "file_type": {
+                                    "title": "File Type",
+                                    "type": "string"
+                                },
+                                "globs": {
+                                    "title": "Globs",
+                                    "type": "array",
+                                    "items": {
+                                        "type": "string"
+                                    }
+                                },
+                                "validation_policy": {
+                                    "title": "Validation Policy",
+                                    "type": "string"
+                                },
+                                "catalog_schema": {
```

Comment:
why is the configured catalog part of the spec?

Reply:
Its here because in the pydantic model for the stream configs the configured catalog has always been there, although @clnoll do you know if this is required for specific functionality. It was in the config since the beginning and never changed this specific field

Reply:
I can't think of a need for it to be in the spec.